pramen-py/tests/conftest.py

```diff
@@ -80,10 +71,33 @@ def create_parquet_data_stubs(
             ],
         ),
     )
+
+
+@pytest.fixture
+def create_data_stubs_and_paths(
+    get_data_stub: DataFrame,
+    tmp_path: pathlib.Path,
+):
+    """Create parquet data stubs partitioned by info_date.
+
+    The path is temporary and is removed after tests finishes.
+    Returns tuple of the path with the parquet data and dataframe itself.
+    """
     table_path = tmp_path / "data_lake" / "example_dependency_table"
-    df.write.partitionBy("info_date").parquet(table_path.as_posix())
-    logger.info("Dataframe successfully created")
-    return table_path, df
+    paths = {"output": (table_path / "output").as_posix()}
```

Comment:
After reading your code, then also previous version, what do you think if we stick to the `input_table` and `output_table` names for the input and output tables paths?